app/controllers/Application.scala

```diff
@@ -26,6 +29,17 @@ class Application(
     Redirect("/uk", request.queryString)
   }
 
+  def geoRedirect: Action[AnyContent] = NoCacheAction() { implicit request =>
+
+    val redirectUrl = request.getFastlyCountry match {
+      case Some(UK) => "/uk"
+      case Some(US) => "/us"
+      case _ => "https://membership.theguardian.com/supporter"
```

Comment:
So we'll just let membership-frontend workout where to send them from here? We do have separate supporter pages for AU/CA & ROW but I'm happy to leave as is.

Reply:
Yeah membership will deal with this